### PR TITLE
Use Red Hat Logo

### PR DESCRIPTION
**Pull request by bshephar**
This changes swaps out the OpenStack logo for the Red Hat one

### CODE REVIEW

**Comment by dprince** [comment]
Kind of sad to see it go. But Lgtm. Thanks @bshephar

**Review by dprince — APPROVED** [review]

File: config/manifests/bases/openstack-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -595,7 +595,7 @@ spec:
...
...vgF2xtBepo4CqMmpUY2/qACaQUUAoAYj3/5FBdAMKgIAFdvw7V9UAG6gIgBQsg3f/kUF4AYqAgAlOjFO4y8qADdREQCY0Mh5//2oABzUVxGcYB0BgDEdGbfxFxWAH1hZDGAM8504PpDnQREAHrFBcNB2D7HpHICE6SWYGbfvP0EAeKobRaaLyPzZH/qzAKAP23PMcyEAPGe3oT7MgDEQrFN2zDA3AqAl7MyhIzYMGCcAwnBG0oE8A79pBEDLME4ABGOifv80AqDFbPeQqQruDv1ZAC10pzmIqshtEQABoHsIaJ17O3E88JzfPAiAwNjZQyYM9oX+LABPfbITx0OPecyDAAhUN4pmbBAcpCoAvDHxjJ9BCIDApQaNqQoAt5Xa+IsAQJqtCg4zVgA4p/TGXwQAhrFjBQeZQQQ0rrQ+/34EAEayM4iSLiLWFQD1KmW2zzAEAMZGFxFQG7PI62DRef4bIQAwkW4UHbSVAXsQAeUy2zsc7sTxQtXPlQBAIalZRIwXAMU9bBv/ifb2yYsAQGlSYcA21UB+lQ32DkMAoBKpwePDrC8ARqqty6cfAYDKpcLgIJUBcI1jnTg+2tQjIQBQK8YMgDWNvfWnEQBoTCoMDrAnEQJhpncerbuvfxgCAM6wU0uTMGDRGdrmlFlQWdcMn3EQAHCSXXR2gHEDtMC8bfgb7e4ZhACA82xX0YFUdxHVAXwwb7t7Kl3NWwQBAO+kqoMDDCTDQc43/AkCAN6zZx8n1QFrDtAU08d/0oeGP0EAoFVS3UUHCATUwMzqMbt1Hu/E8aJvD5wAQKsRCKiImcdvpnLOujSrJy8CAEGxgZAeQ2CGEca1ZBp8X9/2ByEAELy+QeUZZhkhZcU2+l717Y+LAAD6pLqNZlL/yyrlcCRv+nOdOJ5t810TAMAY7IZ2M4RCa82nGn3nFmxVhQAAJmRDYU8qEPYwyOwNM4g7F8Jb/igEAFAyuy4hHQ57GFdolOnHX0g1+K3ry58UAQDUxA4270lVC3voSipdurE3/7vQlhk7VSAAAAfYqkG2Ykimqm6jS2koM1C7aBv5xVRj7+2c/CYQAIAHUgGRBEMSEmppF5Ppo79kG/dF+8+mkb8U0iBt1QgAoEVsN9M2e0dJN5OG/LtqCI+kIU9bSH0sadhF414/AgAAQiTp/wGp07dbIvUIVgAAAABJRU5ErkJggg==
```
Comment (abays) [inline config/manifests/bases/openstack-operator.clusterserviceversion.yaml:598]: There might be an extra whitespace at the end of this line

Comment (bshephar) [reply config/manifests/bases/openstack-operator.clusterserviceversion.yaml]: Indeed

**Review by abays — APPROVED** [review]
/lgtm

**Comment by rh-jelabarre** [comment]
My one question here is how would you check if the logo renders correctly (that there isn't anything wrong in the base64data) without having to do a full deploy?

**Comment by bshephar** [comment]
> My one question here is how would you check if the logo renders correctly (that there isn't anything wrong in the base64data) without having to do a full deploy?

In an automated way? Or manually?

I think it would be difficult to do in an automated way. You would need to render it, then check a specific pixel matches the expected output I guess. But manually is easy:

```
❯ yq '.spec.icon[].base64data' config/manifests/bases/openstack-operator.clusterserviceversion.yaml | base64 -d > test-the-image.png
```

Then just opening the image.

**Comment by bshephar** [comment]
recheck

**Comment by fao89** [comment]
you can copy the CSV here: https://operatorhub.io/preview

**Pull request merged**